no/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Elixir School
+lang: no
+---
+
+[![License](http://img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
+
+Leksjoner i programmeringsspråket Elixir, inspirert av Twitters [Scala Skole](http://twitter.github.io/scala_school/).
+
+
+_Du oppfordres til å delta, samt gi tilbakemeldinger!_
+
+## Om Elixir
+
+"Elixir er et dynamisk, funksjonelt språk utviklet for å bygge skalerbar og vedlikeholdsvennlige applikasjoner." [elixir-lang.org](http://elixir-lang.org/)
```

Comment:
"for å bygge skalerbare og vedlikeholdsvennlige applikasjoner"

Du glemte 'e' i slutten av skalerbar.